tests/nodes/subobjects/test_computation_forcefiled.py

```diff
@@ -0,0 +1,39 @@
+import json
+
+from util import strip_uid_from_dict
+
+import cript
+
+
+def test_computation_forcefield(simple_computation_forcefield_node, simple_computation_forcefield_dict):
```

Comment:
it might be a good idea to put a small docstring in each test to say what it is testing